Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM registry.fedoraproject.org/fedora:29
+
+LABEL maintainer="Factory 2 Team" \
+      description="A microservice triggered by specific message to tag a build."
+
+RUN dnf install -y \
+        python3-pyyaml \
+        python3-fedmsg \
+        python3-koji \
+        python3-requests \
+        python3-qpid-proton \
+        krb5-workstation \
+    && dnf clean all
+
+RUN sed -i '/default_ccache_name = KEYRING:persistent:%{uid}/d' /etc/krb5.conf
+
+WORKDIR /src
+
+COPY . .
+RUN sed -i '/koji/d' requirements.txt
```

Comment:
You should a comment explaining this. I know why we do it because I've run into the same issue, but it's not obvious to someone who's new to the project..